students/biegon_piotr/lesson_05_list_comprehensions/age_calculator.py

```diff
@@ -0,0 +1,20 @@
+avg = 0
+ages = []
+people = int(input("Enter the number of people: "))
+
+for x in range(people):
+    print("Person number", x + 1, "age: ")
+    ages.append(int(input()))
+
+adults = [x for x in ages if x >= 18]
+kids = [x for x in ages if x < 18]
+
+if adults == []:
```

Comment:
the preffered 'pythonic' way to check emptiness would be:
`if not adults:`

> For sequences, (strings, lists, tuples), use the fact that empty sequences are false.

check out more at:
https://stackoverflow.com/questions/53513/how-do-i-check-if-a-list-is-empty